mistralrs-server/src/main.rs

```diff
@@ -89,8 +89,8 @@ struct Args {
     chat_template: Option<String>,
 
     /// Source of the token for authentication.
-    /// Can be in the formats: "literal:<value>", "env:<value>", "path:<value>", "cache" to use a cached token or "none" to use no token.
-    /// Defaults to using a cached token.
+    /// Can be in the formats: `literal:<value>`, `env:<value>`, `path:<value>`, `cache` to use a cached token, or `none` to use no token.
+    /// Defaults to `cache`.
```

Comment:
Switched to backticks instead of double quote wrapping as the CI check considered `<value>` as HTML tag 🤷‍♂️ 

This isn't actually used to generate docs for the crate, only for CLI help. I noticed `mistralrs-server` is locked down heavily on the docs generated, and assume there is a way to exclude it from showing in the generated docs if there's no intended lib usage via the crate.